mim/commands/search.py

```diff
@@ -30,6 +31,7 @@
     'batch_size': 'bs',
     'epochs': 'epoch',
     'inference_time': 'fps',
+    'inference_time_(fps)': 'fps',
```

Comment:
Why both inference_time and inference_time_(fps) maps to fps? Is there any difference between the two names?